src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -77,22 +74,31 @@ export default class GenomicOverview extends React.Component<IGenomicOverviewPro
     }
 
     private shouldShowVAFPlot():boolean {
-        return this.props.mutations.length > 0;
+        return this.props.mergedMutations.length > 0;
     }
 
     private getTracksWidth():number {
         return this.props.getContainerWidth() - (this.shouldShowVAFPlot() ? 140 : 40);
     }
 
-    private computeMutationFrequencyBySample(mutations:Mutation[]):{[sampleId:string]:MutationFrequencies} {
-        const ret:{[s:string]:number[]} = {};
+    private computeMutationFrequencyBySample(mergedMutations:Mutation[][]):MutationFrequenciesBySample {
+        const ret:MutationFrequenciesBySample = {};
         let sampleId;
         let freq;
-        for (let mutation of mutations) {
-            sampleId = mutation.sampleId;
-            freq = mutation.tumorAltCount / (mutation.tumorRefCount + mutation.tumorAltCount);
+        for (let mutations of mergedMutations) {
+            for (const mutation of mutations) {
```

Comment:
why `let mutations` but `const mutation`? Is there a typescript error if you do `const mutations`?